minio/minio
# Add deploymentID to web handler logs

This change makes Web API Handlers log the deploymentID in their logs. 

## Description

The change extends the existing mechanism to add deploymentID and other request related information to the logger entry. 

## Motivation and Context


This would help admins managing logs from multiple MinIO clusters.

## Regression


No

## How Has This Been Tested?



1. Create a Bucket and upload an object to it
2. Open MinIO browser and try deleting the above non-empty bucket
3. MinIO server logs should have an entry like,
```
API: webDeleteBucket()
Time: 14:02:07 PDT 05/29/2019
DeploymentID: b2053bb0-2b9f-49ae-8d7f-0ccc7f4f96f5
RemoteHost: 127.0.0.1
UserAgent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0
Error: Bucket not empty: test
       3: cmd/web-handlers.go:1988:cmd.toWebAPIError()
       2: cmd/web-handlers.go:1868:cmd.toJSONError()
       1: cmd/web-handlers.go:283:cmd.(*webAPIHandlers).DeleteBucket()
```
## Types of changes


## Checklist:
